marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/mapping/mappingWithRelationships/test.sjs

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const test = require("/test/test-helper.xqy");
+
+const esMappingLib = require("/data-hub/5/builtins/steps/mapping/entity-services/lib.sjs");
+let assertions = [];
+
+
+let content = ['/content/mapEntireRecord.json'].map(uri => {
```

Comment:
content is not being used.
Also MapCat step was created but not used in the test.